### PR TITLE
Bump Node.js to v18

### PR DESCRIPTION
**Pull request by austinkelleher**
It is what it says on the box.

### CODE REVIEW

File: packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts [hunk 1]
```diff
@@ -71,13 +53,10 @@ export function registerEventHandlers(
 ): RegisteredEventListeners {
   const multipleResolveListener = createMultipleResolveListener(callback);
   process.on('multipleResolves', multipleResolveListener);
-  const unhandledRejectionListener = createUnhandledRejectionListener(callback);
-  process.on('unhandledRejection', unhandledRejectionListener);
```
Comment (austinkelleher) [inline packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts:75]: The behavior of `unhandledRejection` changed in Node.js 15. I talked with @ndowmon, who implemented this original event handler, and he said that we can remove it.

See: https://developer.ibm.com/blogs/nodejs-15-release-blog/

**Review by VDubber — APPROVED** [review]
LGTM
What are next steps for testing?

File: packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts (hunk 1 above)
Comment (zemberdotnet) [reply packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts:75]: Wdyt about removing multipleResolvesListener as well. It's deprecated.

Comment (austinkelleher) [reply packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts:75]: @ndowmon What do you think?

Comment (ndowmon) [reply packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts:75]: go for it.

**Review by zemberdotnet — APPROVED** [review]

**Pull request merged**